R/rtools-metadata.R

```diff
@@ -56,10 +56,11 @@ version_info <- list(
   "3.4" = list(
     version_min = "3.3.0",
     version_max = "3.4.99",
-    path = if (using_gcc49()) {
-      "bin"
-    } else {
-      c("bin", "gcc-4.6.3/bin")
-    }
+    path = "bin"
+  ),
+  "3.5" = list(
+    version_min = "3.4.0",
```

Comment:
Yes of course, not sure what I was thinking there, I just updated with an open ended version_max.